### PR TITLE
Add CI and finish method of class TaskManager

### PR DESCRIPTION
**Pull request by SergiiTkachuk**

### CODE REVIEW

File: TaskManager.js [hunk 1]
```diff
@@ -119,8 +119,45 @@ class TaskManager {
...
+      });
+   }
+   
+   showPendingTasks() {
 
+      const pendingTasks = this.tasks.filter((task) => !task.completed);
+      
+      pendingTasks.sort((a, b) => {
+         return new Date(a.deadline) - new Date(b.deadline);
+      });
```
Comment (HoloborodkoM) [inline TaskManager.js]: Magic variables 'a' and 'b'

File: TaskManager.js [hunk 2]
```diff
@@ -119,8 +119,45 @@ class TaskManager {
...
    }
-}
 
+   showExpiredTasks() {
+      
+      const currentDate = new Date();
+      const expiredTasks = this.tasks.filter((task) => !task.completed && task.deadline && new Date(task.deadline) < currentDate);
+      
+      console.log('Expired tasks:');
+
```
Comment (HoloborodkoM) [inline TaskManager.js]: Where condition, when empty list for expired tasks?
Think about it. 🤓

File: TaskManager.js [hunk 3]
```diff
@@ -119,8 +119,45 @@ class TaskManager {
...
+   
+   showPendingTasks() {
 
+      const pendingTasks = this.tasks.filter((task) => !task.completed);
+      
+      pendingTasks.sort((a, b) => {
+         return new Date(a.deadline) - new Date(b.deadline);
+      });
+      
+      console.log('Pending tasks (sorted by deadline):');
```
Comment (HoloborodkoM) [inline TaskManager.js]: - Where condition, when empty list for pending tasks?
- If the task doesn't have a field "deadline", what will happen?
Think about it. 🤓

File: TaskManager.js (hunk 1 above)
Comment (SergiiTkachuk) [reply TaskManager.js]: Done

File: TaskManager.js (hunk 2 above)
Comment (SergiiTkachuk) [reply TaskManager.js]: Okey

File: TaskManager.js (hunk 3 above)
Comment (SergiiTkachuk) [reply TaskManager.js]: No problem

**Pull request merged**